Makefile

```diff
@@ -45,3 +45,7 @@ tests: compile
 .PHONY: proto
 proto:
 	script/generate_proto
+
+.PHONY: docker
+docker:
```

Comment:
This target should probably have a `tests` dependency to make sure you have a binary (that passes tests) to copy into the docker container.